src/css/style.css

```diff
@@ -0,0 +1,10 @@
+.header__container {
```

Comment:
@SantFdez there should be a `header` folder for this `container` class

```
|- header/
   |- container/
      |- index.css
   |- mobile-menu-toggle/
      |- index.css
   |- index.css
```

or

```
|- header/
   |- container.css
   |- mobile-menu-toggle.css
   |- index.css
```

and the `header/index.css` should import all styles related to `header/` block